v3.0/reference/configuration/tidb-server/tidb-specific-variables.md

```diff
@@ -1,7 +1,6 @@
 ---
 title: TiDB 专用系统变量和语法
 category: reference
-aliases: ['/docs-cn/sql/tidb-specific/']
```

Comment:
add it back

Reply:
Add back in https://github.com/pingcap/docs-cn/pull/1495/commits/8ea96d204e58ed7a18ca67875b63d679ffba950e